annotations.md

```diff
@@ -40,6 +40,14 @@ This specification defines the following annotation keys, intended for but not l
     ```
 * **org.opencontainers.image.title** Human-readable title of the image (string)
 * **org.opencontainers.image.description** Human-readable description of the software packaged in the image (string)
+* **org.opencontainers.image.base.digest** [Digest](descriptor.md#digests) of the image this image is based on (string)
+  * This SHOULD be the immediate image sharing zero-indexed layers with the image, such as from a Dockerfile `FROM` statement.
+  * This SHOULD NOT reference any other images used to generate the contents of the image (e.g., multi-stage Dockerfile builds).
+* **org.opencontainers.image.base.ref.name** Image reference of the image this image is based on (string)
+  * This SHOULD be image references in the format defined by [distribution/distribution](https://github.com/distribution/distribution/blob/main/reference/reference.go).
+  * This SHOULD be a fully qualified reference name, without any assumed default registry. (e.g., `index.docker.io/some/image` instead of `some/image`).
```

Comment:
add a tag, as the idea is the `base.ref.name` is tag specific, watching for digest changes associated with that tag: `docker.io/org/baseimage:v1`

I'm suggesting a generic `org/baseimage:v1` reference to avoid debates for why `bash`, vs. `alpine`, and which version would seem overly dated years from now.

Reply:
Added a tag.